### PR TITLE
docs: add cypress verify download plugin to the list

### PR DESCRIPTION
**Pull request by elaichenkov**
Adds new [cy-verify-downloads](https://github.com/elaichenkov/cy-verify-downloads) plugin to the plugins list.

### CODE REVIEW

**Comment by elaichenkov** [comment]
Hey, @flotwig @jennifer-shehane @emilyrohrbough @robertguss 
It'd be quite nice if we have any chance to merge it. I'd be grateful, thanks in advance.

**Review by elylucas — DISMISSED** [review]

File: content/_data/plugins.json [hunk 1]
```diff
@@ -603,6 +603,12 @@
           "link": "https://github.com/ling1726/cypress-forced-colors",
           "keywords": ["forced colors", "high contrast", "high contrast mode", "contrast", "testing"],
           "badge": "community"
+        },
+        {
+          "name": "cy-verify-downloads",
+          "description": "Cypress custom command to wait and verify that file was downloaded",
+          "link": "https://github.com/elaichenkov/cy-verify-downloads",
+          "keywords": ["commands", "assertions", "wait", "verify", "download"],
```
Comment (elylucas) [inline content/_data/plugins.json:611]: Hi @elaichenkov,

This json is invalid with a trailing comma here. Also, could you add the community badge ("badge": "community")?

Thanks

**Review by elylucas — CHANGES_REQUESTED** [review]

File: content/_data/plugins.json (hunk 1 above)
Comment (elaichenkov) [reply content/_data/plugins.json:611]: Hey,
Oh, no, sorry. Lost the badge property, after resolving merge conflicts here. I've fixed it.

**Comment by elaichenkov** [comment]
@elylucas can you please review it again?

**Review by elylucas — APPROVED** [review]
thanks!

**Pull request merged**